### ISSUE TITLE
[nnunet/pytorch] benchmark with 2 nodes gives uexpected result

### ISSUE THREAD

**Issue by AwesomePH** [NVIDIA/DeepLearningExamples#1126]
Related to **nnunet/pytorch** 

**Describe the bug**
I ran benchmark on two nodes(8 gpus per node), and I get 16 logs like following:
```
DLL 2022-05-12 14:47:17.492266 - () throughput_train : 194.093  latency_train_mean : 82.435  latency_train_90 : 66.763  latency_train_95 : 72.176  latency_train_99 : 684.991 
208 training, 52 validation, 260 test examples
Filters: [32, 64, 128, 256],
Kernels: [[3, 3, 3], [3, 3, 3], [3, 3, 3], [3, 3, 3]]
Strides: [[1, 1, 1], [2, 2, 2], [2, 2, 2], [2, 2, 2]]
DLL 2022-05-12 14:47:17.490069 - () throughput_train : 194.046  latency_train_mean : 82.455  latency_train_90 : 66.414  latency_train_95 : 69.434  latency_train_99 : 686.963 
208 training, 52 validation, 260 test examples
Filters: [32, 64, 128, 256],
Kernels: [[3, 3, 3], [3, 3, 3], [3, 3, 3], [3, 3, 3]]
Strides: [[1, 1, 1], [2, 2, 2], [2, 2, 2], [2, 2, 2]]
DLL 2022-05-12 14:47:17.492260 - () throughput_train : 194.093  latency_train_mean : 82.435  latency_train_90 : 65.028  latency_train_95 : 69.802  latency_train_99 : 684.95 
208 training, 52 validation, 260 test examples
Filters: [32, 64, 128, 256],
Kernels: [[3, 3, 3], [3, 3, 3], [3, 3, 3], [3, 3, 3]]
Strides: [[1, 1, 1], [2, 2, 2], [2, 2, 2], [2, 2, 2]]
DLL 2022-05-12 14:47:17.490078 - () throughput_train : 194.047  latency_train_mean : 82.454  latency_train_90 : 66.442  latency_train_95 : 67.581  latency_train_99 : 687.056 
```

**To Reproduce**
1. change node default to 2 (or modify benchmark.py to pass node as arg)
2. python scripts/benchmark.py --mode train --gpus 8 --dim 3 --batch_size 2 --results ./results --task 04

**Expected behavior**
I expect the result to be in 1 line, and should be nearly twice the perf of 8 gpus.

**Environment**

* GPUs in the system: (e.g. 2Nodes*(8x A100-SXM-80GB)):
* CUDA driver version (CUDA Version: 11.0 Driver Version: 460.32.03):

**Comment by michal2409** [comment]
We are not supporting multi-node benchmarking at the moment but we will at it in a near future.

**Comment by AwesomePH** [comment]
> 

Thanks. I need to get training performance on multi-node, so I will try to modify some code, can I get some hint where the problem is?

**Comment by AwesomePH** [comment]
> We are not supporting multi-node benchmarking at the moment but we will at it in a near future.

I seems that the `global_batch_size` in `main.py` should be `global_batch_size=batch_size * args.gpus * args.nodes`.

**Comment by michal2409** [comment]
We've pushed the code with support for multi-node benchmarking. Please let me know if it works now.

**Comment by AwesomePH** [comment]
> 

It works for me now.
